filesystems/nfs/connectathon/runtest.sh

```diff
@@ -568,22 +585,16 @@ else
     echo " ========== Use default single test run ============"
 fi
 
-if [ -n "$CTHONSERVERS" ]; then
+if [ -n "$_CTHONSERVERS" ]; then
     # We can specify a list of servers it test against
     # For example, we can provide a customized list of
-    # servers CTHONSERVER="host1-nfs host2-nfs host3-nfs"
-    servers="$CTHONSERVERS"
+    # servers _CTHONSERVER="host1-nfs host2-nfs host3-nfs"
+    servers="$_CTHONSERVERS"
     echo " ========== Override server list =========="
-    echo " ***** INFO: CTHONSERVERS=$CTHONSERVERS *****"
+    echo " ***** INFO: _CTHONSERVERS=$servers *****"
```

Comment:
I wonder if it would be better to rename the variable only in the `if` condition, or maybe even not rename it there either, and simply add a `&& false` or similar at the end. It would be nice to add a comment above the condition, explaining that we're disabling that block temporarily.

Reply:
Hi @emanchado, I use `if false; then ... fi` to disable the related code block and add some comments to explain why, please help to take a look again, thanks!

Reply:
Wait, is that what we want? If the `$CTHONSERVERS` variable *is* defined, won't `servers` end up empty? I was thinking of adding something to the condition to force it to always go to the `else` part...

Reply:
Yes, you're right. Sorry for misunderstanding what you mean. Now I use
```bash
if false && [[ -n "$CTHONSERVERS" ]]; then
    ...
fi
```
instead.